_translator-files/po/de/Software-Manual.po

```diff
@@ -804,19 +804,19 @@ msgstr "Du kannst verschiedene Mixereinstellungen mit [Laden/Speichern Konfigura
 #| msgid ""
 #| "**Note for macOS users:** As of Jamulus 3.8.1, we have a signed installer. This will store the settings in \n"
 #| "```shell\n"
-#| "$HOME/Library/Containers/io.jamulus.Jamulus/Data/.config/Jamulus/\n"
+#| "$HOME/Library/Containers/app.jamulussoftware.Jamulus/Data/.config/Jamulus/\n"
 #| "```\n"
 msgid "**Note for macOS users:** As of Jamulus 3.8.1, we have a signed installer. This will store the settings in \n"
 msgstr ""
 "**Hinweis für macOS-Nutzer:** Ab Jamulus 3.8.1 gibt es ein signiertes Installationsprogramm. Dieser speichert die Einstellungen in\n"
 "```shell\n"
-"$HOME/Library/Containers/io.jamulus.Jamulus/Data/.config/Jamulus/\n"
+"$HOME/Library/Containers/app.jamulussoftware.Jamulus/Data/.config/Jamulus/\n"
```

Comment:
These three copies appear to be duplicates (lines 801 through 820 need cleaning up).

Reply:
I think it’s translation memory. We could clean it but there’s no need.